### ISSUE TITLE
SyntaxError in the example code in readme

### ISSUE THREAD

**Issue by mstfdkmn** [irods/irods_rule_engine_plugin_python#227]
- [x] main
- [x] 4-3-stable

---

It appears the exception syntax in the following line is incorrect: https://github.com/irods/irods_rule_engine_plugin_python/blob/main/README.md?plain=1#L156

When I modified it to `except Exception as e:`, the `check_err` function didn’t work as expected.

**Comment by alanking** [comment]
Good find. To be more specific, here's the Python stacktrace I got when running this against 4.3.3.0 / 4.3.3 on Ubuntu 22.04 (python 3.10.12):
```python
Traceback (most recent call last):
  File \"/etc/irods/core.py\", line 8, in testRule
    if check_err(e, USER_FILE_DOES_NOT_EXIST):
  File \"/etc/irods/core.py\", line 3, in <lambda>
    check_err = lambda e,errcode: e.message.startswith('[iRods__Error__Code:{}]'.format(errcode))
AttributeError: 'RuntimeError' object has no attribute 'message'
```

A `RuntimeError` is being raised and `RuntimeError` does not have a `message` attribute.

In addition to fixing the `SyntaxError`, I made the following adjustment to the example rule and got the expected output:
```diff
3c3
<   check_err = lambda e,errcode: e.message.startswith('[iRods__Error__Code:{}]'.format(errcode))
---
>   check_err = lambda e,errcode: str(e).startswith('[iRods__Error__Code:{}]'.format(errcode))
```
Here's the output with the adjustment:
```
$ irule -r irods_rule_engine_plugin-python-instance -F call_testRule.r '*dataPath="/tempZone/home/rods/goo"'

Level 0: DEBUG: msiObjStat: rsObjStat failed for /tempZone/home/rods/goo, status = -310000
```
We'll get that fixed up
